lib/exercise_test_cases.rb

```diff
@@ -15,6 +16,6 @@ def parsed_json_cases
   end
 
   def to_a
-    parsed_json_cases.map { |test_case| case_classname.new(test_case) }
+    parsed_json_cases.map.with_index { |test_case, index| case_classname.new(test_case, index) }
```

Comment:
I guess it needs to go to skip now, that was one of the nice things about having a `render` method.